dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -87,8 +92,8 @@ def addColumnByFileType(sess, types, fileType, fieldName, fieldNameShort, requir
         newColumn = FileColumn()
         newColumn.file = fileType
         newColumn.required = False
-        newColumn.name = fieldName
-        newColumn.name_short = fieldNameShort
+        newColumn.name = fieldName.lower().strip().replace(' ', '_')
+        newColumn.name_short = fieldNameShort.lower().strip().replace(' ', '_')
```

Comment:
Good question. The initialize script deletes and reloads this every time, so we shouldn't need a data migration.